service/league_service/league_service.py

```diff
@@ -67,7 +68,7 @@ async def update_data(self):
                     .outerjoin(league)
                     .outerjoin(leaderboard)
                 )
-                .where(league_season.c.end_date == None)
+                .where(and_(league_season.c.start_date < datetime.now(), datetime.now() < league_season.c.end_date))
```

Comment:
There might be a "date between" function you can use from sqlalchemy

Reply:
I found a between function, but it doesn't seem to be applicaple, because the dates are in two different columns. Do you mean something else?

Reply:
Have you tried it with putting the columns there? Just looking on stack overflow, idk if this actually works though https://stackoverflow.com/a/6582212

Reply:
Nevermind, I just didn't read correctly